allensdk/brain_observatory/vbn_2022/metadata_writer/dataframe_manipulations.py

```diff
@@ -365,3 +365,80 @@ def _add_images_from_behavior(
             on='ecephys_session_id',
             how='left')
     return ecephys_table
+
+
+def remove_aborted_sessions(
+        lims_connection: PostgresQueryMixin,
+        behavior_df: pd.DataFrame,
+        expected_training_duration: int = 15 * 60,
+        expected_duration: int = 60 * 60
+) -> pd.DataFrame:
+    """
+    Removes aborted sessions
```

Comment:
would be helpful to describe in words the definition of the aborted sessions as listed in the issue, e.g.:
"Any sessions that are shorter than 15 minutes for TRAINING_0 or 1 hour for all other stages can be ignored, as these are not valid."